api/src/main/resources/workbench-api.yaml

```diff
@@ -9015,6 +9023,11 @@ components:
           type: string
           description: |
             Optional, the machine type determines the number of CPUs and memory for the master node. For example "n1-standard-16" or "n1-highmem-64". If unspecified, defaults to creating a "n1-standard-4" machine. To decide which is right for you, see https://cloud.google.com/compute/docs/machine-types
+        zone:
```

Comment:
we should make this required, which will make Typescript enforce its presence.  Unfortunately Java does not enforce it.